fireant/database/vertica.py

```diff
@@ -1,20 +1,25 @@
 # coding: utf-8
-from pypika import terms
+from pypika import terms, VerticaQuery
 
-from . import Database
+from fireant.database import Database
+from fireant.slicer import DatetimeInterval
 
 
 class Trunc(terms.Function):
-    # Wrapper for Vertica TRUNC function for truncating dates.
+    """
+    Wrapper for Vertica TRUNC function for truncating dates.
+    """
 
     def __init__(self, field, date_format, alias=None):
         super(Trunc, self).__init__('TRUNC', field, date_format, alias=alias)
 
 
 class Interval(terms.Interval):
```

Comment:
@twheys @mikeengland Nope. Quarters and years are not properly handled by Pypika's interval, because ANSI SQL treats them as a function of the number of days. Which can vary.